content/en/docs/tasks/traffic-management/ingress/gateway-api/index.md

```diff
@@ -14,17 +14,19 @@ This task describes how to configure Istio to expose a service outside the servi
 These APIs are an actively developed evolution of the Kubernetes [Service](https://kubernetes.io/docs/concepts/services-networking/service/)
 and [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) APIs.
 
-{{< warning >}}
```

Comment:
Do we still need this, or some different, warning now?

Reply:
Can we put it only in the mesh section?

Reply:
Maybe linking to https://gateway-api.sigs.k8s.io/contributing/gamma/ and say its pending upstream agreement

Reply:
Sounds good. I also added a couple more steps to test add header.